ec2/spark_ec2.py

```diff
@@ -505,6 +505,7 @@ def launch_cluster(conn, opts, cluster_name):
         master_group.authorize('tcp', 50070, 50070, authorized_address)
         master_group.authorize('tcp', 60070, 60070, authorized_address)
         master_group.authorize('tcp', 4040, 4045, authorized_address)
+        master_group.authorize('tcp', 8787, 8787, authorized_address)
```

Comment:
Really minor but could you add a comment on the line before saying this is for RStudio ? Might help others figure out why this port is open